Source/Views/ActionSheetView.swift

```diff
@@ -37,6 +37,10 @@ final class ActionSheetView: AlertControllerView {
         self.cancelActionView.layer.cornerRadius = self.visualStyle.cornerRadius
         self.cancelActionView.layer.masksToBounds = true
 
+        if let cancelAction = self.cancelAction {
+            self.cancelButton?.setupAccessibility(using: cancelAction)
```

Comment:
It's on master: https://github.com/sberrevoets/SDCAlertView/blob/master/Source/Views/ActionSheetView.swift#L6